pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -482,6 +485,9 @@ stdenv.mkDerivation (finalAttrs: {
     (enableFeature withVorbis "libvorbis")
     (enableFeature withVmaf "libvmaf")
     (enableFeature withVpx "libvpx")
+    # (enableFeature withVpl "libvpl")
+    # temporary use of optionals to avoid big rebuild
+    ] ++ optionals withVpl [ (enableFeature withVpl "libvpl") ] ++ [
```

Comment:
Nit:

```suggestion
  ] ++ optionals withVpl [
    # temporary use of optionals to avoid big rebuild, removed in staging
    (enableFeature withVpl "libvpl")
  ] ++ [
```

Reply:
And then you'd simply remove 488 and 491 on staging.